projects/packages/forms/src/blocks/contact-form/class-contact-form-block.php

```diff
@@ -71,6 +72,36 @@ public static function render_wrapped_html_block( $content, $parsed_block ) {
 		return $content;
 	}
 
+	/**
+	 * Disable the field inspector tabs for the Contact Form block.
+	 *
+	 * See https://make.wordpress.org/core/2023/03/07/introduction-of-block-inspector-tabs/.
+	 *
+	 * @param array $settings The block editor settings.
+	 * @return array The modified block editor settings.
+	 */
+	public static function disable_field_inspector_tabs( $settings ) {
+		$settings['blockInspectorTabs'] = array_merge(
+			$settings['blockInspectorTabs'] ?? array(),
+			array(
+				'jetpack/field-text'              => false,
+				'jetpack/field-name'              => false,
+				'jetpack/field-email'             => false,
+				'jetpack/field-url'               => false,
+				'jetpack/field-date'              => false,
+				'jetpack/field-telephone'         => false,
+				'jetpack/field-textarea'          => false,
+				'jetpack/field-checkbox'          => false,
+				'jetpack/field-checkbox-multiple' => false,
+				'jetpack/field-radio'             => false,
+				'jetpack/field-select'            => false,
+				'jetpack/field-consent'           => false,
+				'jetpack/field-number'            => false,
+			)
```

Comment:
A hard-coded list like this is not going to be very easily maintainable (if new blocks are added, this is easily missed), but I've done it this way for illustration purposes.

Reply:
I wonder if a little refactor of the contact form field block registrations would help here. If there's a config array used to loop through their registration, the same array can be used to create the block inspector tabs overrides.

It's an implementation detail but for future forms blocks, they'd have to be added to the config array for registration and would then get automatically fed into this.